disnake/enums.py

```diff
@@ -779,6 +779,8 @@ class Locale(Enum):
     "English, US | English, US"
     es_ES = "es-ES"
     "Spanish | Español"
+    es_LATAM = "es-419"
```

Comment:
While perhaps not as clear, I'd prefer to keep the enum names aligned with the table in the API docs by using `es_419` here instead